enterprise-suite/console_test/resources/app_with_multiple_ports.yaml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: es-test-with-multiple-ports
+spec:
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        app: es-test
+      annotations:
+        prometheus.io/scrape: "true"
+    spec:
+      containers:
+        # source: https://github.com/lightbend/k8s-explore/tree/master/query/nan
+        - name: es-test-1
+          image: lightbend-docker-registry.bintray.io/enterprise-suite/es-test:v1
+          imagePullPolicy: IfNotPresent
+          ports:
+            - containerPort: 8080
+              name: metrics
+
+        - name: es-test-2
+          image: lightbend-docker-registry.bintray.io/enterprise-suite/es-test:v1
```

Comment:
Wish I knew where es-test came from... :thinking:

Reply:
A mystery for all of us!